### ISSUE TITLE
Bedre Facebook-overblik ifht. links

### ISSUE THREAD

**Issue by Sobuno** [TK-IT/kasse#30]
![capture](https://user-images.githubusercontent.com/3593183/29463374-bed6a98a-8432-11e7-9916-af14426675bf.PNG)

I ovenstående vil jeg gerne følge med i Betafetas forsøg på at drikke øl. Det er dog lidt uigennemskueligt hvilket af de 4 links, som jeg skal trykke på, for at få hans forsøg frem.

Man kunne inline links ala. nedenstående:

> GFUVE har drukket 3 øl på 0:53.39 (https://tket.dk/5/836), Baaden har drukket 3 øl på 0:28.92 (https://tket.dk/5/838), og Carlsen har drukket 2 øl på 0:32.21(https://tket.dk/5/839).
> Betafeta er i gang med at drikke øl på tid (https://tket.dk/5/837).

Det er dog heller ikke alt for kønt. Alternativt kunne links optræde i samme rækkefølge som deltagerne, så der er en logisk sammenhæng der, eksemplificeret nedenfor:

> GFUVE har drukket 3 øl på 0:53.39, Baaden har drukket 3 øl på 0:28.92, og Carlsen har drukket 2 øl på 0:32.21.
Betafeta er i gang med at drikke øl på tid.
Følg med: https://tket.dk/5/836, https://tket.dk/5/838, https://tket.dk/5/839, https://tket.dk/5/837

**Comment by Mortal** [comment]
Ja, links burde være i samme rækkefølge som personerne er nævnt. Det vil
kræve en smule refactoring af koden. Det er muligt, men det er ikke
trivielt sådan som koden er struktureret nu. Pull requests er velkomne.
